src/core/utils.h

```diff
@@ -21,6 +21,7 @@ inline std::string GameDirectory() {
 }
 
 inline std::string PluginDirectory() { return GameDirectory() + "/addons/counterstrikesharp"; }
```

Comment:
I've seen that it is being used in the dotnet host but this is pretty misleading? I think it should lead to the plugins folder and we should have a different function to get the root of cssharp

Reply:
I mean it refers to the metamod plugin directory but in overall this is misleading and we should have a `GetRootFolder` or something like that

Reply:
Done